config/global.go

```diff
@@ -63,6 +66,14 @@ func (tlc *TopLevelConfig) GetGlobal() (*Global, error) {
 		return nil, err
 	}
 
+	if global.TTL == 0 {
```

Comment:
I think we can safely skip setting defaults here again as `NewGlobalConfig()` already sets the defaults and `json.Unmarshal` will not touch these values if they are not set.